actions/bump-version/action.yaml

```diff
@@ -4,11 +4,13 @@ description: "This action bumps package versions based on the repository's commi
 outputs:
   version:
     description: 'The new version that packages were bumped to.'
+    value: ${{ steps.get_version.outputs.version }}
```

Comment:
I didn't expect to have to do this... but we do: https://docs.github.com/en/actions/sharing-automations/creating-actions/creating-a-composite-action#creating-an-action-metadata-file.